include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -500,6 +500,7 @@ typedef struct OrtOpenVINOProviderOptions {
   size_t num_of_threads;               ///< 0 = Use default number of threads
   unsigned char use_compiled_network;  ///< 0 = disabled, nonzero = enabled
   const char* blob_dump_path;          // path is set to empty by default
+  void* context;
```

Comment:
This is a new member that won't exist in older apps using this structure, how are you ensuring you don't access it if someone is using a previous version of GetApi() and passes in the old version without it?

Reply:
All the values have default parameters , the user only needs to provide the parameters he wishes to
  if (useOPENVINO) {
        // Using OPENVINO backend
        OrtOpenVINOProviderOptions options;
        options.device_type = "GPU_FP32"; //Other options are: GPU_FP32, GPU_FP16, MYRIAD_FP16
        options.context = (void *) ocl_instance->_context.get() ; // If needed 
        std::cout << "OpenVINO device type is set to: " << options.device_type << std::endl;
        sessionOptions.AppendExecutionProvider_OpenVINO(options);
    }

Reply:
If code is rebuilt with the new headers this would be true, but binaries built with the old headers won't be updated.  As they will were compiled against the previous header that didn't have this field.  If we try to read from this field we'll get undefined values or just crash on unreadable memory.

The way we know the user has passed in the new larger version of the structure is based on the version they pass to GetApi(), as this indicates they have the new version of the header.

Reply:
Looks like we don't have any existing binaries with OpenVINO, so this won't be an issue. All good then.

Feel free to resolve this conversation.

Reply:
Sure Ryan